### PR TITLE
CircleCI integration

### PR DESCRIPTION
**Pull request by gggrant**
### Notes:
- `.circleci/config.yml` builds containers, installs dependencies and runs PHPUnit / jest tests.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        environment:
+          PGHOST: 127.0.0.1
+          PGUSER: talentcloud
+      - image: circleci/postgres:9.6-alpine
+        environment:
+          POSTGRES_USER: talentcloud
+          POSTGRES_DB: talentcloud
+    steps: # a set of executable commands
+      - checkout # special step to check out source code to working directory
+      - run: sudo apt update && sudo apt install zlib1g-dev postgresql-9.6 libpq-dev
```
Comment (Khristophor) [inline .circleci/config.yml]: Curious why it's installing postgres inside a postgres:9.6 image?

File: .env.testing [hunk 2]
```diff
@@ -2,7 +2,7 @@ APP_NAME="Talent Cloud"
 APP_ENV=testing
 APP_KEY=base64:lSoNWwEtogJWFNF7ryPiYyXeSdU4ked4NrhRDMvdUFE=
 APP_DEBUG=false
-APP_URL=https://talent.test
+APP_URL=http://localhost
```
Comment (Khristophor) [inline .env.testing]: This might affect local development environments running unit tests

File: .circleci/config.yml (hunk 1 above)
Comment (gggrant) [reply .circleci/config.yml]: Redundant, just needs libpq-dev. Removed.

File: .env.testing (hunk 2 above)
Comment (gggrant) [reply .env.testing]: Also not necessary, switched back.

**Comment by gggrant** [comment]
Not an improvement on our current CI configuration.

**Pull request closed without merging**